tests/part_test.py

```diff
@@ -1320,9 +1320,9 @@ def test_set_part_flags(self):
         self.assertTrue(ps.flags & BlockDev.PartFlag.BOOT)
         self.assertTrue(ps.flags & BlockDev.PartFlag.LVM)
 
-        # SWAP label not supported on the MSDOS table
+        # HPSERVICE flag not supported on the MSDOS table
         with self.assertRaises(GLib.GError):
-            BlockDev.part_set_part_flags (self.loop_dev, ps.path, BlockDev.PartFlag.SWAP)
+            BlockDev.part_set_part_flags (self.loop_dev, ps.path, BlockDev.PartFlag.HPSERVICE)
```

Comment:
So `SWAP` is supported actually?

Reply:
Yes and no. Parted doesn't support swap "flag" on GPT, but there is a GUID for swap partitions and fdisk supports it. I thought it makes sense to support it now, but it might be better to not change the behavior.

Reply:
I wouldn't say it was any well-defined behavior.